python-rag-chatbot/README.md

```diff
@@ -0,0 +1,52 @@
+# Integrating Oracle Database 23ai RAG and OCI Generative AI with LangChain
+
+[**Oracle Database 23ai**](https://www.oracle.com/database/free-1/)
+
+[**OCI GenAI**](https://www.oracle.com/artificial-intelligence/generative-ai/large-language-models/)
+
+[**LangChain**](https://www.langchain.com/) 
+ 
+
+## TODO instructions
+
+- setup ~/.oci/config
+- set yourcompartmentid
+- podman run -d --name 23ai -p 1521:1521 -e ORACLE_PWD=Welcome12345 -v oracle-volume:/Users/pparkins/oradata container-registry.oracle.com/database/free:latest
```

Comment:
Just checking this password isn't real - it's OK as long as we don't show any IP address or associated endpoint 👍🏻